juju/juju
# Handling docker resource from CLI.

## Description of change

This adds support in Juju for the docker resource type (in charms).

Currently it is a WIP and needs further work.

## QA steps

Using custom charms, ensure that they deploy correctly (i.e. use the resources as they should be).
Further solid QA steps to come in the lifetime of this WIP.

## Documentation changes

There will need to be some help documentation changes to the deploy command.
There will need to be some help documentation changes to the help regarding charms and resource types.
